### PR TITLE
fix: update multicall with .env.bridge file

### PR DESCRIPTION
**Pull request by lesterli**
## Summary

This PR updates the scripts under `l2-bridge` to fix an issue: 
```
Updating .env.bridge file for OP Bridge UI...
/home/snapchain/op-chain-deployment/scripts/l2-bridge/l2-bridge-set-env.sh: line 37: NEXT_PUBLIC_L1_MULTICALL3_ADDRESS: unbound variable
make[1]: *** [Makefile:86: l2-bridge-start] Error 1
```

we removed `NEXT_PUBLIC_L1_MULTICALL3_ADDRESS` from the `.env` file, so:

* `l2-bridge-deploy-l1-multicall.sh`: for the multicall was deployed case, it should update`.env.bridge`;
* `l2-bridge-set-env.sh`: remove replacement line for `NEXT_PUBLIC_L1_MULTICALL3_ADDRESS`

**Updates**: Also, it will add two env vars for bridge ui.

## Test Plan

To change the `L1_RPC_URL` with the Sepolia, run:
```
make l2-bridge-deploy-l1-multicall 
```
it should update `NEXT_PUBLIC_L1_MULTICALL3_ADDRESS` in `.env.bridge` with the value `0xcA11bde05977b3631167028862bE2a173976CA11`

### CODE REVIEW

File: scripts/l2-bridge/l2-bridge-deploy-l1-multicall.sh [hunk 1]
```diff
@@ -6,33 +6,42 @@ set -a
...
+DEPLOYED_URL=$(curl -s https://raw.githubusercontent.com/mds1/multicall/main/deployments.json | jq -r ".[] | select(.chainId == $CHAIN_ID) | .url")
 if [ ! -z "${DEPLOYED_URL}" ]; then
     # Extract the Ethereum address from the URL
+    echo "Extracting multicall contract address from the deployed URL..."
     DEPLOYED_ADDRESS=$(echo "$DEPLOYED_URL" | grep -oE '0x[a-fA-F0-9]{40}')
     if [ -n "$DEPLOYED_ADDRESS" ]; then
         echo "Contract is already deployed. Exiting."
-        # Update the .env file with the deployed address
-        sed -i.bak "s/^NEXT_PUBLIC_L1_MULTICALL3_ADDRESS=.*/NEXT_PUBLIC_L1_MULTICALL3_ADDRESS=${DEPLOYED_ADDRESS}/" ../.env
-        rm ../.env.bak
```
Comment (bap2pecs) [inline scripts/l2-bridge/l2-bridge-deploy-l1-multicall.sh:22]: i am a bit confused. why simply changing these two lines to .env.bridge won't work?

Comment (lesterli) [reply scripts/l2-bridge/l2-bridge-deploy-l1-multicall.sh:22]: actually it should work, and other changes are the improvements and logs

Comment (bap2pecs) [reply scripts/l2-bridge/l2-bridge-deploy-l1-multicall.sh:22]: got it. yeah i think we shouldn't touch .env for bridge related scripts.

it's cleaner to just use .env.bridge

**Comment by bap2pecs** [comment]
lgtm

**Pull request merged**